Added support for Redis v2.8.x.

tjanez opened the pull request:
Ansible checks if the version of Redis is 2.8.x and uses a different
configuration template if it is.

tjanez committed:
Added support for Redis v2.8.x.

Ansible checks if the version of Redis is 2.8.x and uses a different
configuration template if it is.

bennojoy commented:
Hi ,

Could you please split the task into two , eventhough the below is a valid jinja, we dont encourage usning conditionals within the {{ }}.
- name: Copy the Redis configuration file 
-  template: src={{ "redis-2.8.conf.j2" if redis_version.stdout | search("v=2.8") else "redis.conf.j2" }} dest={{ redis_conf_dest }}

so the task woudl have 

 template: src=redis-2.8.conf.j2
when:redis_version.stdout | search("v=2.8")

Similarly for the other pr also please.

tjanez commented:
Sure, no problem. I'll update both PRs.

tjanez committed:
Splitted the task that copies the configuration file into two.

This was done to avoid using Jinja2's conditionals in the playbook, which is
discouraged.

tjanez commented:
@bennojoy, I updated the PR. Please, review.

tjanez commented:
@bennojoy, ping?
